bin/ci.sh

```diff
@@ -11,7 +11,7 @@ typecheck() {
 lint() {
   REPO_URL="https://raw.githubusercontent.com/ripple/javascript-style-guide"
   curl "$REPO_URL/es6/eslintrc" > ./eslintrc
-  echo "parser: esprima-fb" >> ./eslintrc
+  echo "parser: babel-eslint" >> ./eslintrc
```

Comment:
This supports facebook flow (vaguely recall someone telling me that's what esprima-fb was for )  
?